Assets/HoloToolkit/Sharing/Tests/Scripts/RemoteHeadManager.cs

```diff
@@ -137,7 +138,7 @@ private void UpdateHeadTransform(NetworkInMessage msg)
             Quaternion headRot = CustomMessages.Instance.ReadQuaternion(msg);
 
             RemoteHeadInfo headInfo = GetRemoteHeadInfo(userID);
-            headInfo.HeadObject.transform.localPosition = headPos;
+            headInfo.HeadObject.transform.localPosition = headPos += new Vector3(0,-1.0f,0);
```

Comment:
I'm still learning vectors, but does this mean the avatar is 1 meter below the position of the hololens?